micromamba_env.yml

```diff
@@ -0,0 +1,8 @@
+name: mgrowthdb
+channels:
+  - conda-forge
+  - anaconda
+  - bioconda
```

Comment:
Actually not sure which of these channels are necessary, I'll reset my env at some point and rebuild it to test.